search/searcher/search_fuzzy.go

```diff
@@ -31,9 +34,10 @@ func NewFuzzySearcher(indexReader index.IndexReader, term string,
 			break
 		}
 	}
-
+	t := time.Now()
 	candidateTerms, err := findFuzzyCandidateTerms(indexReader, term, fuzziness,
```

Comment:
i think this is why fuzzy is slower, you're finding the candidate terms the old way even when you later use the push-down capability

Reply:
Ya as @sreekanth-cb pointed out, disregard this comment.  I misread the code due to the way github folded it in the diff.